Resources/doc/commands.md

```diff
@@ -0,0 +1,68 @@
+# Commands
+
+## Remove cache
+
+
+All cache for a given paths will be lost:
+
+``` bash
+$ php app/console liip:imagine:cache:remove path1 path
+```
+
+
+If you use --filters parameter, all cache for a given filters will be lost:
+
+``` bash
+$ php app/console liip:imagine:cache:remove --filters=thumb1 --filters=thumb2
+```
+
+You can combine these parameters:
+
+``` bash
+$ php app/console liip:imagine:cache:remove path1 path2 --filters=thumb1 --filters=thumb2
+```
+
+Cache for all paths and filters will be lost when executing this command without parameters :
+
+``` bash
+$ php app/console liip:imagine:cache:remove
```

Comment:
@makasim makes me think we should confirm in case someone provides no path or filter that its not accidentally